packages/wonder-blocks-core/components/view.js

```diff
@@ -25,7 +25,18 @@ const styles = StyleSheet.create({
     },
 });
 
+type ValidViewTags = "div" | "article" | "aside" | "nav" | "section";
+// eslint-disable-next-line flowtype/require-exact-type
+type Props = {
+    ...TextViewSharedProps,
+    tag?: ValidViewTags,
```

Comment:
We should make this required here and the set `"div"` to be the default value using `defaultProps`.